packages/jaeger-ui/src/components/TracePage/TraceSpanView/index.tsx

```diff
@@ -205,7 +205,11 @@ export default class TraceSpanView extends Component<Props, State> {
                 }}
               >
                 {this.state.serviceNamesList.map(name => {
-                  return <Option key={name}>{name} </Option>;
+                  return (
+                    <Option value={name} key={name}>
+                      {name}{' '}
```

Comment:
just curious, what's the need for the extra space in the display string?

Reply:
Previously it was `return <Option key={name}>{name} </Option>;` (Notice an extra space after `{name}`).
So, on running `yarn fmt`, prettier detected that maybe that space is needed, and formatted it that way (`{' '}`)